include/micro_ros_utilities/type_utilities.h

```diff
@@ -32,18 +32,61 @@ extern "C"
 // Memory rule
 typedef struct micro_ros_utilities_memory_rule_t
 {
+  /**
+   * Pattern matching the fully qualified name of a msg field.
+   *
+   */
   const char * rule;
+
+  /**
+   * Memory (in bytes) to allocate for fields matching \ref rule.
+   *
+   */
   size_t size;
 } micro_ros_utilities_memory_rule_t;
 
 // Memory configuration string
 typedef struct micro_ros_utilities_memory_conf_t
 {
+  /**
+   * Maximum string capacity to use for msg fields in case they don't have a
+   * custom rule assigned to them.
+   *
+   */
   size_t max_string_capacity;
+
+  /**
+   * Maximum capacity to use for sequence type msg fields (ie: unbounded
+   * arrays and lists) which contain ROS 2 msg types, in case they don't have
+   * a custom rule assigned to them.
+   *
+   */
   size_t max_ros2_type_sequence_capacity;
+
+  /**
+   * Maximum capacity to use for sequence type msg fields (ie: unbounded
+   * arrays and lists) which contain basic types (ie: primitive field types),
+   * in case they don't have a custom rule assigned to them.
+   *
+   */
   size_t max_basic_type_sequence_capacity;
+
+  /**
+   * All ruled defined in this configuration.
```

Comment:
```suggestion
   * Custom rules defined in this configuration.
```

Reply:
Does a `micro_ros_utilities_memory_conf_t` contain non-custom rules by default?

If it doesn't, explicitly mentioning *custom rules* here would imply there could be more rules, they just wouldn't be custom rules.

Reply:
Fine by me, I have updated the change to fix a typo.